test_aodndata/aodn_wave_dm/test_aodn_wave_dm.py

```diff
@@ -100,16 +100,18 @@ def test_dest_path_aodn_wave_dm(self):
         self.assertEqual(dest_path_aodn_wave_dm(good_nc),
                          os.path.join(
                              'IMOS/NTP/Low_Cost_Wave_Buoy_Technology',
+                             'Waverider_Buoys',
                              'DELAYED',
-                             'TOR01',
+                             'Torbay01',
                              os.path.basename(good_nc)))
 
         good_nc = os.path.join(TEST_ROOT, 'IMOS_NTP-WAVE_W_20200113T215850Z_TOR01_WAVERIDER_FV01_timeseries_END-20200319T015105Z.nc')
```

Comment:
sorry I've just noticed that the 2 tests on good files are testing the same feature on 2 different files. One test of this kind is sufficient, so you can delete either one of these.